module/library/categories/helper.js

```diff
@@ -213,14 +214,25 @@ module.exports = class LibraryCategoriesHelper {
 				let result = await projectTemplateQueries.getAggregate(aggregateData)
 
 				if (result[0].data.length > 0) {
-					result[0].data.forEach((resultedData) => {
+					for (const resultedData of result[0].data) {
```

Comment:
@praveenKDass lets use foreach

Reply:
As we are using await to getDownloadble Url ,forEach will not works as expected @VISHNUDAS-tunerlabs , So changed it to for loop